keyboards/crkbd/keymaps/via/rules.mk

```diff
@@ -1,3 +1,2 @@
-MOUSEKEY_ENABLE = no     # Mouse keys
```

Comment:
This is likely required to keep the firmware size down on older revisions.

Reply:
disable mousekey at the json file https://github.com/qmk/qmk_firmware/pull/22676/commits/e70e4b2ecbddcb9cc8d2d92d365bcfe75522c0a4